packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -370,12 +372,13 @@ export default defineComponent({
 			}
 
 			const { ownedBy, sharedWith, ...credentialData } = this.credentialData;
-			const hasExpressions = Object.values(credentialData).reduce(
+			const hasUntestableExpressions = Object.values(credentialData).reduce(
 				(accu: boolean, value: CredentialInformation) =>
-					accu || (typeof value === 'string' && value.startsWith('=')),
+					accu ||
+					(typeof value === 'string' && isExpression(value) && !isTestableExpression(value)),
```

Comment:
@valya I think you worked on this part. Can you check and answer please?

Reply:
It needs to be an `and` since the true case causes it to not be tested. Either way, it should be working now. There was a backend issue but I've sorted it now